### ISSUE TITLE
Hunt-http does not compile on linux .

### ISSUE THREAD

**Issue by kerdemdemir** [huntlabs/hunt-http#13]
I wanted to use web sockets but dependency hunt-web did not exists. 
Than I cloned hunt-web and run dub build myself but it seems hunt-http is not compiling neither. 

Ubuntu 18.04 
DMD64 D Compiler v2.089.0
hunt-web = master 
hunt-http = 0.3 
Error : 
erdem@erdem-HP-EliteDesk-800-G2-TWR ~/dev/hunt/hunt-web (master) $ dub build
Performing "debug" build using /usr/bin/dmd for x86_64.
hunt-web 1.0.0-beta.1: building configuration "default"...
../../../.dub/packages/hunt-http-0.3.0/hunt-http/source/hunt/http/client/package.d(1,1): Error: module hunt.http.client from file ../../../.dub/packages/hunt-http-0.3.0/hunt-http/source/hunt/http/client/package.d conflicts with package name client
/usr/bin/dmd failed with exit code 1.

Erdem

**Comment by zoujiaqing** [comment]
You can using latest master, only dependency hunt-http ( remove hunt-web libaray).

**Comment by kerdemdemir** [comment]
Good news !! 

Library compiles !! 

But now examples are not compiling. It fails in linking stage. I have ssl-dev already but does not helps. 


(dmd-2.089.1)erdem@erdem-GE60-2PC:~/dev/hunt2/hunt-http/examples/HttpDemo$ ./build.sh 
Performing "debug" build using dmd for x86_64.
hunt 1.4.3: building configuration "library"...
hunt-openssl 1.0.1: building configuration "library"...
hunt-net 0.4.5: building configuration "ssl"...
hunt-http 0.5.0-rc.1+commit.1.gb0e1e16: building configuration "tls"...
http-demo ~master: building configuration "client"...
Linking...
/home/erdem/.dub/packages/hunt-net-0.4.5/hunt-net/.dub/build/ssl-debug-linux.posix-x86_64-dmd_2089-52630812BC4F56DD4ECBB151C49BCC6E/libhunt-net.a(safestack_176a_449.o): In function `_D6deimos7openssl9safestack__T10SKM_sk_numTSQBqQBm3ssl13ssl_cipher_stZ__TQBrZQBvFNbPSQDfQDbQCw__T8STACK_OFTQCmZQoZi':
/home/erdem/.dub/packages/hunt-openssl-1.0.1/hunt-openssl/deimos/openssl/safestack.d:140: undefined reference to `OPENSSL_sk_num'
/home/erdem/.dub/packages/hunt-net-0.4.5/hunt-net/.dub/build/ssl-debug-linux.posix-x86_64-dmd_2089-52630812BC4F56DD4ECBB151C49BCC6E/libhunt-net.a(safestack_176c_516.o): In function `_D6deimos7openssl9safestack__T12SKM_sk_valueTSQBsQBo3ssl13ssl_cipher_stZ__TQBtZQBxFNbPSQDhQDdQCy__T8STACK_OFTQCmZQoiZPQCv':
/home/erdem/.dub/packages/hunt-openssl-1.0.1/hunt-openssl/deimos/openssl/safestack.d:142: undefined reference to `OPENSSL_sk_value'
collect2: error: ld returned 1 exit status
Error: linker exited with status 1

**Comment by kerdemdemir** [comment]
Finally I got the examples compiled as well !! . 

I had to download and install openssl 1.1 manually . 

Also make if DMD64 D Compiler v2.089.0 is used compiling fails with  

/usr/include/dmd/druntime/import/core/atomic.d(246,23): Error: undefined identifier U
/home/erdem/.dub/packages/hunt-1.4.3/hunt/source/hunt/concurrency/atomic/AtomicHelper.d(58,41): Error: template instance core.atomic.atomicExchange!(cast(MemoryOrder)5, IForkJoinTask, shared(IForkJoinTask)) error instantiating
/home/erdem/.dub/packages/hunt-1.4.3/hunt/source/hunt/concurrency/ForkJoinPool.d(3357,43):        instantiated from here: getAndSet!(IForkJoinTask, IForkJoinTask)

But if you upgrade to DMD64 D Compiler v2.089.1 it does not fails. 

Thanks a lot. I think issue can be closed.
